features/steps/cli_steps.py

```diff
@@ -174,48 +172,6 @@ def exec_kedro_target_checked(context, command):
         assert False
 
 
-@given('I have created new environment "{}"')
```

Comment:
There were some steps not used in any tests, found using `behave -f steps.usage --dry-run features/`. I've removed them but let me know if there's some reason we want to keep them.